docusearch/management/commands/import_documents.py

```diff
@@ -51,6 +57,9 @@ def convert_to_text(file_name, doc_path):
 
 
 def create_document(document, release_slug):
+    """ Create a Document object representing the document. This also uploads
```

Comment:
When I looked at this function the first time around I had to a bit of searching to figure out how the documents were uploaded to s3. Maybe add a bit about `django-storages` here or in the model.